Components/Distributions/Distribution.gd

```diff
@@ -33,9 +33,10 @@ func pick():
 	if cumulative_elements[HIGH_INDEX].p < p:
 		LOW_INDEX = HIGH_INDEX
 	element = cumulative_elements[LOW_INDEX]
-	if is_distribution(element.value):
-		return element.value.pick()
```

Comment:
Is there any reason this isn't an array too?

Reply:
I'm not sure what you mean - what variable or thing do you think should / could be an array?

Reply:
Don't worry, looking at it again it shouldn't be :)